res_users_password_security/README.rst

```diff
@@ -0,0 +1,94 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================
+Password  Security
+==================
+
+This module allows admin to set company-level password security requirements
+and enforces them on the user.
+
+It contains features such as
+
+* Password expiration days
+* Password length requirement
+* Password minimum number of lowercase letters
+* Password minimum number of uppercase letters
+* Password minimum number of numbers
+* Password minimum number of special characters
```

Comment:
I guess I've only seen crap implementations like Magento then 😛 . 

This make sense though, so basically instead of trying to reference hash vs hash, I need to validate the non-hashed passwords against the previous X. Piece of cake, thanks @hbrunn